dev/bots/prepare_package.dart

```diff
@@ -685,14 +715,14 @@ Future<void> main(List<String> rawArguments) async {
   }
 
   final String revision = parsedArguments['revision'] as String;
-  if (revision.isEmpty) {
+  if (revision == null) {
```

Comment:
Actually, I think it would be better to use `parsedArguments.wasParsed('revision')` (and below), since that is really asking whether or not they supplied it.

Reply:
ack

Reply:
done